lib/getNextJobs.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/**
+ * Calculate the next jobs to execute, given a workflow and a trigger job
+ * @method getNextJobs
+ * @param  {Object}    workflow         Directed graph representation of workflow
```

Comment:
Can you rename this variable to `workflowGraph` so we don't get it mixed up with the old concept of `workflow`?